src/engine/strat_engine/filesystem.rs

```diff
@@ -99,6 +99,13 @@ impl Filesystem for StratFilesystem {
             Err(e) => Err(EngineError::Engine(ErrorEnum::Error, e.description().into())),
         }
     }
+
+    fn devnode(&self) -> EngineResult<PathBuf> {
```

Comment:
we have `impl From<devicemapper::DmError> for EngineError` that should make it happy?